api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/CreateCardJob.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.dhsc.htbhf.claimant.scheduler;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.JobExecutionContext;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@AllArgsConstructor
+@DisallowConcurrentExecution
+public class CreateCardJob extends QuartzJobBean {
```

Comment:
I think it would be worthwhile some JavaDoc here to describe why we're using `@DisallowConcurrentExecution`.

Reply:
Done.